packages/electric/lib/templates/api-component.tpl

```diff
@@ -7,7 +7,7 @@ import Soy from 'metal-soy';
 import '<%= component %>';
 <% }); %>
 
-import 'electric-marble-components/lib/ElectricAPIEntities.soy.js';
+import '<%= apiEntitiesPath %>.soy.js';
```

Comment:
I'm wondering if we should include the `.soy` in there as well. Does the `soyName` right below include it?

Reply:
Hey @jbalsas it only includes `index.soy.js`. It seems to work fine so I do not think it's necessary to include `.soy` since it has compiled it.

Does the metal work with `.soy` or just `.soy.js`?

Reply:
Hey @matuzalemsteles, I mean that we down there we have:

```javascript
import templates from './<%= soyName %>.js';
```

Which I assume we're using like `soyName = index.soy`.

If that's the case, we have some consistency problems:
- We likely want this new variable to start with `soy` to make it clear this is a template
- We need to distinguish between `*Name` and `*Path`

So, my proposals would be:

```javascript
// Proposal 1 -> soyAPIEntitiesPath = "electric-marble-components/lib/ElectricAPIEntities.soy.js"
import '<%= soyAPIEntitiesPath %>';

// Proposal 2 -> soyAPIEntitiesName = "electric-marble-components/lib/ElectricAPIEntities.soy"
import '<%= soyAPIEntitiesName %>.js';
```

I think I'd rather go with `1`, since it's more generic and doesn't impose any other assumptions, specially since the path might be outside the project and in the `node_modules` dependencies.

What do you think?

Reply:
Oh, had not understood. It's clearer now 🙂.

Yes, I would also go with Proposal 1, it seems more generic.

I'll send this. Thanks @jbalsas.